src/tab-container-element.ts

```diff
@@ -117,6 +130,13 @@ export class TabContainerElement extends HTMLElement {
     this.#setup = true
```

Comment:
Not related to this PR but wondering if we can improve on the naming here. setupComplete fully communicated the intent here, for a moment I had to double check we were finished with setup when this is true, not that it meant that we were _doing_ setup (if that makes sense). Again, non-blocking thoughts.